setup.php

```diff
@@ -1289,14 +1289,27 @@ function cloneClicked() {
                         }
 
                         $short_tag = ini_get('short_open_tag') ? 'On' : 'Off';
+                        $short_tag_style = $short_tag.strcmp('Off') ? '' : 'text-danger';
                         $display_errors = ini_get('display_errors') ? 'On' : 'Off';
+                        $display_errors_style = $display_errors.strcmp('Off') ? '' : 'text-danger';
                         $register_globals = ini_get('register_globals') ? 'On' : 'Off';
+                        $register_globals_style = $register_globals.strcmp('Off') ? '' : 'text-danger';
                         $max_input_vars = ini_get('max_input_vars');
+                        // $max_input_vars_style = $max_input_vars.strcmp('at least 3000') ? '' : 'text-danger';
+                        $max_input_vars_style = $max_input_vars < 3000 ? 'text-danger' : '';
+
                         $max_execution_time = ini_get('max_execution_time');
+                        // $max_execution_time_style = $max_execution_time.strcmp('at least 60') ? '' : 'text-danger';
+                        $max_execution_time_style = $max_execution_time < 60 ? 'text-danger' : '';
```

Comment:
hi @stu01509 , note that `0` is also ok here since it means no time limit. I wouldn't document that since would be confusing, but would not highlight this as incorrect if it is `0`.

Reply:
OK, Thanks for your explanation.